src/assets/styles/header.scss

```diff
@@ -293,21 +293,22 @@
   .toolbar-inner {
     display: flex;
     justify-content: center;
-    align-items: stretch;
   }
 
   .toolbar-item {
     margin: 0 ($offset / 2);
-    display: flex;
-    flex-direction: column;
 
-    > * {
-      flex: 1;
+    .btn,
```

Comment:
Just took out the attempts at flexboxing the height of these items, ie was barfing on it. This is not a great solution, it is brittle, but it works.